### DISCUSSION TITLE
On Blocking user my activity on github deleted, Unable to get it back now after unblocking

### DISCUSSION THREAD

**Discussion by rudrakshkarpe** [community/community#35169] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/78851635/194004509-5ff1c13d-54cb-4d44-8a60-30c27bbfb3b1.png)

**Comment by rudrakshkarpe** [comment]
@github-staff please help me fix this issue, I'm having a huge loss my on GitHub activity

**Comment by jfullstackdev** [comment]
that's an intended behavior

![image](https://user-images.githubusercontent.com/47092464/194093768-bacd2d13-d99b-4889-ad9c-95877662887b.png)

https://docs.github.com/en/communities/maintaining-your-safety-on-github/blocking-a-user-from-your-personal-account#about-blocking-users

if you are not willing to unblock the user again, you might want to try contacting Support, maybe they can do something about that, but there is no assurance they will help you

https://support.github.com/contact/account

**Reply by rudrakshkarpe** [reply]
Thanks, I got through the same process as of now there's not any response from GitHub support the initial responses I received were from bots
